orttraining/orttraining/python/training/ortmodule/_inference_manager.py

```diff
@@ -67,10 +67,6 @@ def forward(self, *inputs, **kwargs):
             # If model was exported, then initialize the graph builder
             self._initialize_graph_builder(training=False)
 
-            # Save the onnx model if the model was exported
-            if self._save_onnx:
```

Comment:
Do not remove any existing model

Reply:
This is not removed. It is done in the ```_export_model``` step for both training and inference. So, it is not needed here again.